docker/copy-embedded-settings

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eux
+/bin/mkdir -p /etc/iml
+service=iml_influxdb
+trailer=$(/usr/bin/docker service ps -f "name=$service.1" $service -q --no-trunc | head -n1)
+until /usr/bin/docker exec $service.1.$trailer /bin/bash -c 'until [ -f /var/lib/chroma/iml-settings.conf ]; do echo "Waiting for settings."; sleep 1; done'; do
```

Comment:
Paranoia: is writing the config file atomic? 🤔

Reply:
Yes https://github.com/whamcloud/integrated-manager-for-lustre/blob/docker-config/chroma_core/lib/service_config.py#L782-L784